internal/pkg/build/sources/conveyorPacker_oci.go

```diff
@@ -408,7 +408,7 @@ func (cp *OCIConveyorPacker) insertEnv() (err error) {
 		if len(envParts) == 1 {
 			export = fmt.Sprintf("export %s=${%s:-}\n", envParts[0], envParts[0])
 		} else {
-			if envParts[0] == "PATH" {
+			if strings.ContainsAny(envParts[0],"PATH") {
```

Comment:
I think you would want to use:  `strings.Contains()` instead of `ContainsAny()` as well ... `ContainsAny()` seems to have different behavior, and `Contains()` looks more like what you want for looking for substrings.

This change is going to take *any* variable that includes `PATH` and go down this code path isn't it? So, if I have a variable named:  `PATHOS`, or `SOCIOPATH` ... then it will be caught and go down this path right?

Reply:
`ContainsAny` will simply look for charset.
`Contains` looks for substrings. So yes, `SOCIOPATH` will match as well.

Reply:
Right, but if you do:  `ContainsAny(blah, "ABCD")`

Does that look for `ABCD` or look for *any* A, B, C, or D?

Reply:
As implied from the name, any of the chars. See examples here: https://golang.org/pkg/strings/#ContainsAny